docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rhel7/rhel
+
+LABEL org.feedhenry.fh-template-apps-version="5.0.13" \
+      org.feedhenry.fh-template-apps-build-number="747"
```

Comment:
Would these be required if the image produced just contains template apps, the tag on the image can indicate all this like all other components e.g. docker.io/rhmap/feedhenry-sdks:5.0.13-747

There would be extra work here to keep these values in sync, but i'm not sure they offer anything anyway,